x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/format_parameter.tsx

```diff
@@ -38,9 +39,15 @@ export const FormatParameter = ({ defaultValue, defaultToggleValue }: Props) =>
       title={i18n.translate('xpack.idxMgmt.mappingsEditor.formatParameter.fieldTitle', {
         defaultMessage: 'Set format',
       })}
-      description={i18n.translate('xpack.idxMgmt.mappingsEditor.formatParameter.fieldDescription', {
-        defaultMessage: 'The date format(s) that can be parsed.',
-      })}
+      description={
+        <FormattedMessage
+          id="xpack.idxMgmt.mappingsEditor.formatParameter.fieldDescription"
+          defaultMessage="The date format(s) that can be parsed. Most of the built-in formats have a {strict} companion format, which means year, month, and day must use 4, 2, and 2 digits, respectively."
```

Comment:
@gchaps I'd like to hear what you think here. When I read "which means year, month, and day must use 4, 2, and 2 digits, respectively" I had to do some mental translation to associate 4 with year, 2 with month, and 2 with day. I wonder if there's a way to reword this so that this translation isn't necessary? Maybe something like:

> which means year must use 4 digits, month must use 2 digits, and day must use 2 digits.

Reply:
I agree, I think you suggestion is a little easier to parse.

Reply:
How about showing the format:

The date formats to parse.  Most builit-ins use `strict` date formats, where YYYY is the year, MM is the month, and DD is the day. Example: 2020/11/01.

Or:

The date formats to parse.  Most built-in formats use a `strict` date format: 4 digits for year, 2 digits for month, and 2 digits for day.

Does this text still work with the hint text below the field?

Reply:
I went with your first suggestion:

<img width="709" alt="Screen Shot 2020-01-08 at 9 45 03 AM" src="https://user-images.githubusercontent.com/5226211/71987436-2cb80b00-31fc-11ea-8be0-6a1242866a48.png">